src/ajax.js

```diff
@@ -796,13 +796,14 @@ jQuery.each( [ "get", "post" ], function( i, method ) {
 			data = undefined;
 		}
 
-		return jQuery.ajax({
+		//extend to handle situation when url is actually a config object
```

Comment:
I'd rather leave it since when I started reading jquery source this would have confused me quite a bit. I don't feel strongly about it though if you think I should remove it.